acquire/utils.py

```diff
@@ -291,8 +295,7 @@ def check_and_set_acquire_args(
 
         # If initialization of the plugin fails, a ValueError is raised
         upload_plugin = upload_plugin_cls(**args.config)
-
-    setattr(args, "upload_plugin", upload_plugin)
+        setattr(args, "upload_plugin", upload_plugin)
```

Comment:
why is this set two times?

Reply:
It needs to be set (and set to a default value), as the upload code checks on this parameter.